generate.sh

```diff
@@ -1,84 +1,14 @@
 # #!/bin/bash
 set -euo pipefail
 
-output_file="passage.gen.go"
+if [ -z "$1" ]; then
+  echo "Required generator file is missing."
+  exit 1
+fi
 
-# Skip pointer on optional fields
-fields="\
-.components.schemas.CreateUserRequest.properties.email \
-.components.schemas.CreateUserRequest.properties.phone \
-.components.schemas.CreateUserRequest.properties.user_metadata \
-.components.schemas.UpdateUserRequest.properties.email \
-.components.schemas.UpdateUserRequest.properties.phone \
-.components.schemas.UpdateUserRequest.properties.user_metadata \
-.components.schemas.CreateMagicLinkRequest.properties.channel \
-.components.schemas.CreateMagicLinkRequest.properties.email \
-.components.schemas.CreateMagicLinkRequest.properties.language \
-.components.schemas.CreateMagicLinkRequest.properties.magic_link_path \
-.components.schemas.CreateMagicLinkRequest.properties.phone \
-.components.schemas.CreateMagicLinkRequest.properties.redirect_url \
-.components.schemas.CreateMagicLinkRequest.properties.send \
-.components.schemas.CreateMagicLinkRequest.properties.ttl \
-.components.schemas.CreateMagicLinkRequest.properties.type \
-.components.schemas.CreateMagicLinkRequest.properties.user_id \
-.components.schemas.MagicLinkType \
-.components.schemas.MagicLinkChannel \
-.components.schemas.UserInfo.properties.user_metadata \
-.components.schemas.UserInfo.properties.webauthn_types"
-for field in $fields; do
-  jq "$field |= . + {\"x-go-type-skip-optional-pointer\": true}" openapi.json > tmp.json && mv tmp.json openapi.json
-done
-
-# Rename component to avoid name clash with generated struct
-jq ".components.schemas.ListPaginatedUsersResponse |= . + {\"x-go-name\": \"PaginatedUsersResponse\"}" openapi.json > tmp.json && mv tmp.json openapi.json
-
-# JSON string of key-value pairs
-transforms='{
-    "Active": "StatusActive",
-    "CreateMagicLinkRequest": "CreateMagicLinkBody",
-    "CreateUserRequest": "CreateUserBody",
-    "Inactive": "StatusInactive",
-    "MagicLinkTypeLogin": "LoginType",
-    "MagicLinkTypeVerifyIdentifier": "VerifyIdentifierType",
-    "MagicLinkChannel": "ChannelType",
-    "MagicLinkChannelEmail": "EmailChannel",
-    "MagicLinkChannelPhone": "PhoneChannel",
-    "Pending": "StatusPending",
-    "UpdateUserRequest": "UpdateBody",
-    "UserInfo": "User",
-    "UserMetadataFieldTypeBoolean": "BooleanMD",
-    "UserMetadataFieldTypeDate": "DateMD",
-    "UserMetadataFieldTypeEmail": "EmailMD",
-    "UserMetadataFieldTypeInteger": "IntegerMD",
-    "UserMetadataFieldTypePhone": "PhoneMD",
-    "UserMetadataFieldTypeString": "StringMD"
-}'
-
-# Function to perform replacements
-command -v gorename &> /dev/null || go install golang.org/x/tools/cmd/gorename@v0.24.0
-replace() {
```

Comment:
the reason for wanting to drop this is so the `generate sdk` job for go stops failing due to rename reference failures.

i think the job should be considered successful if the codegen itself succeeds, regardless of whether the hand-written sdk code complies.

and since we always review the codegen PRs anyways, that might be a better place for us to review the hand-written code failures from the codegen.